frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-items.tsx

```diff
@@ -248,6 +296,19 @@ const determineAvailableFilters = (initialFilters, items, filterGroups) => {
           });
         }
       }
+
+      if (value !== undefined && Array.isArray(value)) {
```

Comment:
Is `value !== undefined` needed? `Array.isArray(undefined)` is still `false`.

Reply:
Gotcha